validphys2/src/validphys/checks.py

```diff
@@ -21,6 +21,12 @@
 import logging
 log = logging.getLogger(__name__)
 
+@make_check
+def check_use_t0(ns, **kwargs):
+    """Checks use_t0 is set to true"""
+    if not ns.get("use_t0"):
+        raise CheckError("The flag 'use_t0' needs to be set to 'true' for this action.")
+
```

Comment:
Is this the correct behavior? It checks for this `ns` object which I think is just an object of the Config class that is constructed manually as opposed to taken directly in from the runcard (meaning setting `use_t0` flag in the runcard as suggest in the comments does not work).

Reply:
@scarlehoff

Reply:
isn't `use_t0` one of the runcard flags? (where are you getting the error to understand the problem?)

Reply:
That's what I understood when reading the comments in this PR. But `ns` does not contain all runcard keys. 

A minimal example to get the error is below:

runcard:
```
use_t0: true

actions_:
  - foo
```

with vp action: 
```
@check_use_t0
def foo():
    pass
```

Reply:
I think `check_use_t0` is only supposed to be used in functions that take `use_t0` as input (so with `foo()` it will never find it, but with `foo(use_t0)` will only accept to run `foo` with `use_t0=True`.

Maybe since we added `use_t0_in_fitting` and `_in sampling` (or similar) this is creating a problem somewhere?

Reply:
What function in particular is being problematic? `use_t0_in_fitting/sampling` should only be used during the fit and have no effect if you are not fitting.

Reply:
I want to run a closure test but can't. `internal_multiclosure_data_loader` gives the error. 

They shouldn't affect anything, but perhaps something was touched that broke this down the line. On the other hand, CT worked in Michaels branch, but I never used it after it had been merged so maybe it never worked on the main branch...